### PR TITLE
Added Resources jdbc repositories tests

### PR DESCRIPTION
**Pull request by iri989**

### CODE REVIEW

File: cosmic-usage-api/src/test/resources/test-public-ips-repository-data.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+INSERT INTO `domain`
```
Comment (eliasgomes) [inline cosmic-usage-api/src/test/resources/test-public-ips-repository-data.sql]: I'd suggest renaming this file to `public-ips-repository-test-data.sql` and doing the same with other similar files like `test-domains-repository-data.sql`, `test-virtual-machines-repository-data.sql`, etc.

File: cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/ResourcesJdbcRepositoryIT.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("local")
+public class ResourcesJdbcRepositoryIT {
```
Comment (eliasgomes) [inline cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/ResourcesJdbcRepositoryIT.java]: I'd suggest having individual tests for each repository. That means splitting `ResourcesJdbcRepositoryIT` into `VirtualMachinesJdbcRepositoryIT`, `VolumesJdbcRepositoryIT`, etc.

**Review by eliasgomes — CHANGES_REQUESTED** [review]

**Comment by borisroman** [comment]
go build

**Comment by borisroman** [comment]
go build

File: cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/PublicIpsJdbcRepositoryIT.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("local")
+public class PublicIpsJdbcRepositoryIT {
+
+    @Autowired
+    private PublicIpsJdbcRepository publicIpsRepository;
```
Comment (eliasgomes) [inline cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/PublicIpsJdbcRepositoryIT.java]: We shouldn't lock the test to any specific implementation of a repository, JDBC in this case. I would suggest instead to point to the repository's interface (`PublicIpsRepository`), rename the test class to `PublicIpsRepositoryIT` and move the class one package up.

File: cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/VirtualMachinesJdbcRepositoryIT.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("local")
+public class VirtualMachinesJdbcRepositoryIT {
+
+    @Autowired
+    private VirtualMachinesJdbcRepository virtualMachinesRepository;
```
Comment (eliasgomes) [inline cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/VirtualMachinesJdbcRepositoryIT.java]: Please see my comment in `PublicIpsJdbcRepositoryIT.java`

File: cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/VolumesJdbcRepositoryIT.java [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("local")
+public class VolumesJdbcRepositoryIT {
+
+    @Autowired
+    private VolumesJdbcRepository volumesRepository;
```
Comment (eliasgomes) [inline cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/VolumesJdbcRepositoryIT.java]: Please see my comment in `PublicIpsJdbcRepositoryIT.java`

File: cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/PublicIpsJdbcRepositoryIT.java (hunk 3 above)
Comment (iri989) [reply cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/PublicIpsJdbcRepositoryIT.java]: Done! Thanks :)

**Review by eliasgomes — APPROVED** [review]

**Pull request merged**